packages/ove-app-networks/README.md

```diff
@@ -41,3 +41,13 @@ The networks app has a transparent background. If required, a background colour
 ## Controlling the App
 
 The controller of the app can be loaded by accessing the URL `http://OVE_APP_NETWORKS_HOST:PORT/control.html?oveSectionId=SECTION_ID`.
+
```

Comment:
We should state in the documentation that filters are chained (because we use statements like `f = f.nodesBy(...)`), and this has the effect of combining applied filters with a logical AND.

There should also be an API call to clear all filters.